From bb6af3b07f6ef66d193bdee0345cc2b3a782e8b3 Mon Sep 17 00:00:00 2001
From: Yu Watanabe <watanabe.yu+github@gmail.com>
Date: Tue, 23 May 2023 06:03:52 +0900
Subject: [PATCH] core/unit: drop doubled empty line

(cherry picked from commit 512df9de23890fcfd5fdbfe633250fe848195d4b)

Related: #2213521
---
 src/core/unit.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/core/unit.c b/src/core/unit.c
index 0d1a590a3f..0faf66413b 100644
--- a/src/core/unit.c
+++ b/src/core/unit.c
@@ -987,7 +987,6 @@ static int unit_per_dependency_type_hashmap_update(
         if (r < 0)
                 return r;
 
-
         return 1;
 }
 
